internal/e2e/e2e_rank_test.go

```diff
@@ -311,7 +311,7 @@ func marshalMatches(w io.Writer, rq rankingQuery, q query.Q, files []zoekt.FileM
 		chunks, hidden := splitAtIndex(f.ChunkMatches, chunkMatchesPerFile)
 
 		for _, m := range chunks {
-			_, _ = fmt.Fprintf(w, "%d:%s%s\n", m.ContentStart.LineNumber, string(m.Content), addTabIfNonEmpty(m.DebugScore))
+			_, _ = fmt.Fprintf(w, "%d:%s%s", m.ContentStart.LineNumber, string(m.Content), addTabIfNonEmpty(m.DebugScore))
```

Comment:
could you rather trim the trailing newline here? Otherwise when we enable debug scoring it will print very weirdly

Reply:
Yeah, can do.

Reply:
Done in https://github.com/sourcegraph/zoekt/pull/747/commits/1f9b00fb1d410f439bb4c2b76133fcfd0e2bc99c